### PR TITLE
Add Repair EU Logo

### PR DESCRIPTION
**Pull request by jordycosta**
This pull request adds the logo from https://repair.eu/ to _react-commerce_ to be used in the sub-module on carpenter-frontend

QA: A quick preview to ensure that nothing is broken on page load, no new console errors
_(The changes where this logo is implemented will be in https://github.com/iFixit/ifixit/pull/47751)_

Thank you @mmarcon91 for helping me with this svg

### CODE REVIEW

**Comment by deltuh-vee** [comment]
QA 🎬 
* Nothing is broken on page load and there are no new console errors

File: packages/footer/components/SocialMedia.tsx [hunk 1]
```diff
@@ -21,12 +22,21 @@ export interface SocialMediaAccounts {
...
    accounts: SocialMediaAccounts;
+   repairUrl?: string;
 }) {
    if (!accounts) {
       return null;
    }
+
+   let showRepairEU = false;
+
+   if (repairUrl && repairUrl.includes('eu')) {
```
Comment (aburke07) [inline packages/footer/components/SocialMedia.tsx:36]: We can just move this check inline with the `icon={showRepairEU ? RepairEULogo : RepairOrgLogo}` and get rid of this unneeded variable.

**Review by aburke07 — DISMISSED** [review]
CR :alien: 

dev_block :alien: on some easy cleanup

**Comment by jordycosta** [comment]
un_dev_block 🟢

**Review by aburke07 — APPROVED** [review]
CR :alien:

**Comment by k0rvusk0r4x** [comment]
Still working without errors.

QA 😺

**Pull request merged**